release notes/v0.27.0.md

```diff
@@ -0,0 +1,137 @@
+- The new `--execution-segment` and `--execution-segment-sequence` options allow easy partitioning of test runs across multiple k6 instances. Initially this applies to configuration options like number of VUs or iterations, but opens the door to test data partitioning, an often requested feature. See [#997](https://github.com/loadimpact/k6/issues/997) for more details.
+- Different executors can execute different functions other than the `default` exported one. This can be specified by the `exec` option in each `scenarios` config, and allows for more flexibility in organizing your tests, easier code reuse, building test suites, etc.
+- To allow for even greater script flexibility and code reuse, you can specify different environment variables and tags in each scenario, via the new `env` and `tags` executor options respectively.
+- k6 will now emit a new `dropped_iterations` metric in `shared-iterations`, `per-vu-iterations`, `constant-arrival-rate` and `ramping-arrival-rate` executors if it can't schedule an iteration on time, depending on the configured rates and the iteration duration ([#1529](https://github.com/loadimpact/k6/pull/1529)).
+
+
+## UX
+
+- CLI: There are separate descriptions and real-time thread-safe progress bars for each individual executor.
+- A new method to stop engine execution via the REST API ([#1352](https://github.com/loadimpact/k6/issues/1352)). Thanks @hynd!
+
+## Bugs fixed!
+- CLI: Stop `--http-debug` from exiting k6 on request dump error ([#1402](https://github.com/loadimpact/k6/issues/1402)). Thanks @berndhartzer!
+- CLI: JSON output is now less noisy ([#1469](https://github.com/loadimpact/k6/issues/1469)). Thanks @michiboo!
+- CLI: k6 doesn't exit when using `iterations` with `stages` ([#812](https://github.com/loadimpact/k6/issues/812)).
+- CLI: Mismatch in check counts in the end-of-test summary ([#1033](https://github.com/loadimpact/k6/issues/1033)).
+- Config: Better validation of `stages` ([#875](https://github.com/loadimpact/k6/issues/875)).
+- JS: Rare panics in goja ([#867](https://github.com/loadimpact/k6/issues/867)).
+- HTTP: Fix request timeout and wrong context used for pushing metrics ([#1260](https://github.com/loadimpact/k6/issues/1260)).
+- Execution: Fix sometimes skipping iterations with a `context cancelled` error when rapidly ramping up and down ([#1283](https://github.com/loadimpact/k6/issues/1283)).
+- WebSockets: Fix possible connection hanging when the main context is cancelled ([#1260](https://github.com/loadimpact/k6/issues/1542)).
```

Comment:
maybe also mention that:
1. now there are less goroutine leaks around 1542 and https://github.com/loadimpact/k6/pull/1513
2. that we've updated the ws lib which should add up to some minuscule performance increase  https://github.com/loadimpact/k6/pull/1515

also the number of the PR is wrong ;)

Reply:
https://github.com/loadimpact/k6/pull/1367 this also hasn't been in a release

Reply:
See 6236f14b.

I put the WS lib update under "Internals" since it's not really a bug fix.

> also the number of the PR is wrong ;)

I would rather link to issues than PRs, as those usually describe the problem, and sometimes link to other related issues. Except when the PR doesn't have an issue, but we we should always try to create one for tracking and discussion purposes.